webapp/channels/src/components/header_footer_route/header.tsx

```diff
@@ -39,7 +39,7 @@ const Header = ({alternateLink, backButtonURL, onBackButtonClick}: HeaderProps)
     }
 
     return (
-        <div className='hfroute-header'>
+        <div className={'hfroute-header' + (freeBanner ? ' has-free-banner' : '') + (title ? ' has-custom-site-name' : '')}>
```

Comment:
nit: using the `classnames` utility here would allow to have something a bit more readable:

```tsx
className={classNames('hfroute-header', {'has-free-banner': freeBanner, 'has-custom-site-name': title})}
```

Reply:
Thanks for the guidance. Fixed in the latest commit